doc/cephfs/fs-volumes.rst

```diff
@@ -1046,6 +1046,120 @@ following command.
 
 .. _subvol-pinning:
 
+Controlling Subvolume Snapshot Visibility
+-----------------------------------------
+
+.. note:: This functionality is currently supported only for FUSE clients.
+          Kernel client support is planned; progress can be tracked at
+          https://tracker.ceph.com/issues/72589.
+
+Snapshots of a subvolume can be hidden from compatible clients by
+performing two actions:
+    1) Disable the subvolume's ``snapshot_visibility`` flag (default is ``true``).
+    2) Enable the client-side configuration option ``client_respect_subvolume_snapshot_visibility``
+       for the intended client(s).
+
+The CLI command for toggling ``snapshot_visibility`` is as follows:
+
+.. prompt:: bash #
+
+   ceph fs subvolume snapshot_visibility set <vol_name> <sub_volname> [--group-name <subvol_group_name>] <true|false>
+
+This command updates the internal vxattr ``ceph.dir.subvolume.snaps.visible``
+and sets the ``is_snapdir_visible`` flag within the dirinode(i.e. subvolume’s)
+SnapRealm.
+
+.. note:: Although direct modification is possible, using the subvolume API is
+          recommended. It is more convenient and avoids potential ``EPERM``
+          (Permission Denied) errors when setting vxattrs, especially if the
+          client lacks the required capabilities. The way to set this vxattr
+          is:
+
+          .. prompt:: bash #
+
+             setfattr -n ceph.dir.subvolume.snaps.visible -v 0|1 <subvolume_path>
+
+The ``client_respect_subvolume_snapshot_visibility`` setting determines whether
+a client honors the subvolume's visibility flag. This can be set per-client
+using:
+
+.. prompt:: bash #
+
+   ceph config set client.<id> client_respect_subvolume_snapshot_visibility <true|false>
+
+.. note:: The `id` over here is a CephX user.
+
+To set the ``client_respect_subvolume_snapshot_visibility`` config globally
+i.e. across all the clients then set the config without mentioning the ``id``:
+
+.. prompt:: bash #
+
+   ceph config set client client_respect_subvolume_snapshot_visibility <true|false>
+
+.. note:: Changing the global config value won't apply the changes to the
+          CephFS instance running inside MGR daemon i.e all the ``ceph fs subvolume snapshot``
+          commands would be executed according to MGR's config. To apply
+          changes to MGR's CephFS instance, please make use of:
```

Comment:
There's a CephFS instance in the Manager?

Suggest s/MGR/the Mananger daemons/

I suspect what's intended is something like:

.. note:: Changing the central config client option does not affect behavior within
          the cluster's Manager daemons.  For example, ``ceph fs subvolume snapshot``
          commands use central config values set with  ``mgr`` for the ``who`` scope. To apply
          changes to Managers and associated CLI commands, please issue a command of the following form:


That said, might we make this simpler by advising that the admin set the option at ``global`` scope?